docs/plugins.md

```diff
@@ -390,3 +390,50 @@ Leaflet keeps it simple. If you can think of a feature that is not required by a
 There are no hard requirements on how to create your own plugin, but all developers are encouraged to read the recommendations in the [plugin guide](https://github.com/Leaflet/Leaflet/blob/main/PLUGIN-GUIDE.md).
 
 Once your plugin is ready, you can submit it: just send a pull request with a new plugin file in [/docs/_plugins/](https://github.com/Leaflet/Leaflet/tree/main/docs/_plugins)to our GitHub repository.
+
+<script>
+function loadRepoData() {
+	const rows = document.querySelectorAll('.plugins table tr');
+	rows.forEach((row) => {
+		try {
+			const repoData = row.querySelector('.repo-data');
+			if (repoData) {
+				const link = row.querySelector('.plugin-repo-url').href;
+				let badges = [];
+				if (link.indexOf('github.com') > -1) {
+					const repo = link.split('github.com/')[1];
+					badges = [
+						`https://badgen.net/github/stars/${repo}`,
+						`https://badgen.net/github/last-commit/${repo}`
+					];
+				}
+				if (link.indexOf('github.io') > -1) {
+					const linkData = link.split('.github.io/');
+					const user = linkData[0].split('//')[1];
```

Comment:
I'd go fancier with a regexp like `^https?:\/\/(\w+)\.github\.io\/(\w+)\/` - would be a bit more robust than relying on splitting strings and wrapping in a `try`-`catch` block.

```
const regexpGithubIO = /^https?:\/\/([\w\d-_]+)\.github\.io\/([\w\d-_]+)\/?/;

let match = link.match(regexpGithubIO);
if (match) {
    const repo = `${match[1]}/${match[2]}`;
    badges = /* etc */
}
```

Reply:
`http://leaflet.github.io/leaflet.draw/` would not work currently, should we add `.` too? `^https?:\/\/([\w\d-_]+)\.github\.io\/([\w\d-_\.]+)\/?`

I didn't added the `try-catch` for catching errors while splitting. I added it to make sure if any error happens, that we continue with the next row instead of stopping completely.

Reply:
Should be `www` added too? `/^https?:\/\/(?:www)?\.github`

Reply:
> should we add `.` too? `^https?:\/\/([\w\d-_]+)\.github\.io\/([\w\d-_\.]+)\/?`

Yeah, looks good.

> Should be `www` added too? `/^https?:\/\/(?:www)?\.github`

No, since https://www.ivansanchez.github.io/Leaflet.Marker.Stack/demos/simple.html doesn't seem to be the same as https://ivansanchez.github.io/Leaflet.Marker.Stack/demos/simple.html .